unstructured_inference/models/__init__.py

```diff
@@ -3,6 +3,8 @@
 
 from unstructured_inference.models.detectron2 import load_model, Detectron2LayoutModel
 
+OUTPUT_DIR = "yolox_outputs/"
```

Comment:
Since this constant gets used in `yolox_model.py`, I put the constant in `yolox_model.py` (if we wind up keeping the constant)